core-web/apps/dotcms-ui/src/app/app-routing.module.ts

```diff
@@ -92,17 +92,19 @@ const PORTLETS_ANGULAR: Route[] = [
     },
     {
         path: 'edit-page',
-        canActivate: [editPageGuard],
+        canMatch: [editPageGuard],
         loadChildren: () =>
             import('@portlets/dot-edit-page/dot-edit-page.module').then((m) => m.DotEditPageModule)
     },
     {
-        path: 'edit-ema',
+        path: 'edit-page',
         resolve: {
             data: (route: ActivatedRouteSnapshot) => {
                 return inject(EmaAppConfigurationService).get(route.queryParams.url);
             }
         },
+        // WE NEED TO DELETE THIS WHEN ARCADIO FNISHES THE WORK ON THE CIRCULAR DEPENDECIES, REMIND ME OF THIS IF YOU READ THIS.
+        // eslint-disable-next-line @nx/enforce-module-boundaries
```

Comment:
@oidacra how this is going?

Reply:
This is the PR https://github.com/dotCMS/core/pull/27991 there are missing 2-3. But One of them `DotPushPublishEnvironmentsResolver` has a strange behavior. And the module missing is `DotAddToBundleModule`

Reply:
Fixed. https://github.com/dotCMS/core/pull/27991
Migration done!

The problem is to pass a module/component to UI and use DotTest. I deleted the dependency in DotAddToBundleModule because the test is marked as skipped, and I did not fix it.

Reply:
I forced a commit deleting all the enforce boundaries comments. We should wait for https://github.com/dotCMS/core/pull/27991 to merge this PR.

cc: @fmontes

Reply:
Fixed test issues in the #27991 issue.